### DISCUSSION TITLE
User groups - why enterprise only?

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#13001] [General]
It feels like this is a feature that should just be part of the premium package? 
It would make user management so much easier and I'd curtaining be onboarding more users if it was easier, but stepping up to enterprise level is just not going to happen for me.

Please consider making this part of premium :)

**Comment by melohagan** [comment]
Hey @andyburgessmd 

We believe user groups provide most value for large enterprises with many users.

You can still manage individual user roles, including custom roles, on the premium tier.

If you'd like to discuss an Enterprise agreement, please drop an email to **sales@budibase.com** and we'd be happy to chat through your specific use-case.

**Comment by andyburgessmd** [comment]
@melohagan already heard the minimums for enterprise from Joe and we're not going to get near that.

Roles have their use and are fine in an application but doesn't help when trying to onboard or manage user application access. When someone new needs adding I need to add them as a user to each of the applications. Groups seems to be the only way to do that. 
We were looking at adding a whole group of customers to access some applications but the time it will take to do that is quite an overhead. Why can't you make that a bit easier and encourage us to use more users = more revenue for you! Everyone wins!
